src/components/composite/Header.stories.js

```diff
@@ -29,11 +28,35 @@ const props = {
   },
 };
 
-storiesOf('Composite|Header', module)
-  .addParameters({ component: Header })
-  .add('default', () => <Header {...props} />)
-  .add('inverted', () => (
-    <BlueBackground>
-      <Header isInverted {...props} />
-    </BlueBackground>
-  ));
+export default {
+  component: Header,
+  title: 'Composite/Header',
+};
+
+export const Basic = args => <Header {...args} />;
+Basic.storyName = 'default';
+Basic.args = {
```

Comment:
Redundant with `props`?

Reply:
This was fixed in a prior commit. Thanks for the time reviewing it.